<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideShowWidget</class>
 <widget class="QWidget" name="SlideShowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slideshow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="fullScreen">
     <property name="text">
      <string>Switch to &amp;full-screen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="startWithCurrent">
     <property name="text">
      <string>S&amp;tart with current image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>De&amp;lay between slides:</string>
     </property>
     <property name="buddy">
      <cstring>delayTime</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="delayTime">
     <property name="specialValueText">
      <string>Wait for key</string>
     </property>
     <property name="suffix">
      <string> sec</string>
     </property>
     <property name="maximum">
      <number>3600</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Iterations (0 = infinite):</string>
     </property>
     <property name="buddy">
      <cstring>cycles</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="cycles">
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
